FRAuthenticator/FRAuthenticator/Push/FRAPushHandler.swift

```diff
@@ -260,11 +266,19 @@ public class FRAPushHandler: NSObject {
     
     
     /// Updates all current device tokens with designated AM instance using information given in QR code
+    ///
+    /// NOTE: The device token is automatically stored within the method `application(_ application:, didRegisterForRemoteNotificationsWithDeviceToken deviceToken:)`.
+    /// Use this method for manual updates. The device token provided will overwrite any previously stored token.
```

Comment:
Same here. The wording seems off. Should it be "The _**provided**_ device token will..."?